autorest/responder_test.go

```diff
@@ -198,7 +202,8 @@ func TestByCopying_AcceptsNilBody(t *testing.T) {
 
 func TestByClosing(t *testing.T) {
 	r := mocks.NewResponse()
-	err := Respond(r, ByClosing())
+	err := Respond(r,
+		ByClosing())
```

Comment:
I had added `ByDiscardingBody()` while on auto-pilot applying my fix, but thought better of it because this test should isolate `ByClosing()`. I'll fix up this formatting so that it doesn't jump out to folks looking at diffs later.